src/C-interface/csr/bml_copy_csr_typed.c

```diff
@@ -136,7 +136,11 @@ void TYPED_FUNC(
 {
     const int NNZ = arow->NNZ_;
     // Check size for data
-    assert(brow->alloc_size_ >= NNZ);
+//    assert(brow->alloc_size_ >= NNZ);
```

Comment:
Could you also remove this line? I don't see a reason to keep it.

Reply:
Got it.